### PR TITLE
[KO] Translation of Basics lesson

### PR DESCRIPTION
**Pull request by devleoper**
Related: #201 

If you want to review or compare this with English text, devleoper/elixir-school#3 could be helpful.

### CODE REVIEW

**Comment by devleoper** [comment]
@Dalgona @rijgndqw012 It's ready for review!

**Comment by doomspork** [comment]
Thank you @devleoper for all your hard work 👍

**Comment by rijgndqw012** [comment]
You did very well. :+1:

File: ko/lessons/basics/basics.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+4
+iex> 2 - 1
+1
+iex> 2 * 5
+10
+iex> 10 / 5
+2.0
+```
+
+정수로 된 몫이나 나머지를 구하고 싶을 떼 Elixir에 내장된 함수 두 개를 유용하게 사용할 수 있습니다.
```
Comment (riseshia) [inline ko/lessons/basics/basics.md]: `싶을 떼` -> `싶을 때`

Comment (devleoper) [inline ko/lessons/basics/basics.md]: @riseshia 늦어서 죄송합니다. 반영했습니다 👍

**Pull request merged**